modules/nf-core/bamtofastq10x/main.nf

```diff
@@ -0,0 +1,54 @@
+process BAMTOFASTQ10X {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/10x_bamtofastq:1.4.1--hdbdd923_2':
+        'biocontainers/10x_bamtofastq:1.4.1--hdbdd923_2' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("${meta.id}/**/*.fastq.gz"), emit: fastq
```

Comment:
changed the path from `path("*.fastq.gz")`

Reply:
I wonder if the behavior for bamtofastq is different for cellranger multi and cellranger BAMs? The module has test cases and it seems fastq files were written directly to the main directory in that case. Maybe `**/*.fastq.gz` would work as a generic pattern that works in all cases (to be updated in the central module).

Reply:
Implemented `**/*.fastq.gz` as suggested.